app/models/APIModel.scala

```diff
@@ -3,13 +3,13 @@ package models
 case class Status(service: String, environment: String, serverTime: String) {
 
   object Status {
-    def apply(service: String, environment: String, serverTime: String): Status ={
-      var status = new Status(service, environment, serverTime)
-      status
+    def apply(service: String, environment: String, serverTime: String) ={
     }
   }
 }
```

Comment:
So generally you'll see the object and the case class on the same file but their definitions not nested in each other. Additionally, the `apply()` method here can be ignored (you should be able to just instantiate `Status(...// etc)` without needing to explicitly define this apply (it comes with the case class definition)